sanity_check.sh

```diff
@@ -5,21 +5,22 @@ SCRIPTS='./bessctl/conf/samples'
 OUTFILE='sanity_check.out'
 
 TESTS='exactmatch.bess
-	flowgen.bess
-	generic_encap.bess
-	hash_lb.bess
-	igate.bess
-	iplookup.bess
-	l2_forward.bess
-	multicore.bess
-	queue.bess
-	roundrobin.bess
-	s2s.bess
-	tc/complextree.bess
-	unix_port.bess
-	update.bess
-	vlantest.bess
-	wildcardmatch.bess'
+flowgen.bess
+generic_encap.bess
+hash_lb.bess
+igate.bess
+iplookup.bess
+l2_forward.bess
+multicore.bess
+queue.bess
+reattachment.bess
```

Comment:
Good point. I will do this. Why don't I just put the reattachment bit inside wildcardmatch.bess?